examples/OTV-Ozage-Tang-vortex-2D/input_test.i3d

```diff
@@ -0,0 +1,133 @@
+! -*- mode: f90 -*-
+
+!===================
+&BasicParam
+!===================
+
+! Flow type (1=Lock-exchange, 2=TGV, 3=Channel, 4=Periodic hill, 5=Cylinder, 6=dbg-schemes)
+itype = 2
+
+! Domain decomposition
+p_row=0               ! Row partition
+p_col=0               ! Column partition
+
+! Mesh
+nx=128                ! X-direction nodes
+ny=128                ! Y-direction nodes
+nz=1                  ! Z-direction nodes
+istret = 0            ! y mesh refinement (0:no, 1:center, 2:both sides, 3:bottom)
+beta = 0.259065151    ! Refinement parameter (beta)
+
+! Domain
+xlx = 6.28318530717958 ! Lx (Size of the box in x-direction)
+yly = 6.28318530717958 ! Ly (Size of the boy in y-direction)
+zlz = 6.28318530717958 ! Lz (Size of the boz in z-direction)
+
+! Boundary conditions
+nclx1 = 0 
+nclxn = 0 
+ncly1 = 0 
+nclyn = 0 
+nclz1 = 0 
+nclzn = 0 
+
+! Flow parameters
+iin = 1               ! Inflow conditions (1: classic, 2: turbinit)
+re = 50.             ! nu=1/re (Kinematic Viscosity)
+u1 = 8.               ! u1 (max velocity) (for inflow condition)
+u2 = 8.               ! u2 (min velocity) (for inflow condition)
+init_noise  = 0.0     ! Turbulence intensity (1=100%) !! Initial condition
+inflow_noise = 0.0    ! Turbulence intensity (1=100%) !! Inflow condition
+
+! Time stepping
+dt = 0.0001            ! Time step
+ifirst = 1             ! First iteration
+ilast = 100            ! Last iteration
+
+! Enable modelling tools
+ilesmod=0             ! if 0 then DNS
+iscalar=0             ! If iscalar=0 (no scalar), if iscalar=1 (scalar)
+iibm=0                ! Flag for immersed boundary method
+
+! Enable io
+ivisu=1               ! Store snapshots
+ipost=1               ! Do online postprocessing
+
+ifilter=0
+C_filter=0.49
+
+mhd_active = .true.
+/End
+
+!====================
+&NumOptions
+!====================
+
+! Spatial derivatives
+ifirstder = 4         ! (1->2nd central, 2->4th central, 3->4th compact, 4-> 6th compact)
+isecondder = 5        ! (1->2nd central, 2->4th central, 3->4th compact, 4-> 6th compact, 5->hyperviscous 6th)
+
+! Time scheme
+itimescheme = 5       ! Time integration scheme (1->Euler,2->AB2, 3->AB3, 4->AB4,5->RK3,6->RK4)
+
+/End
+
+!=================
+&InOutParam
+!=================
+
+! Basic I/O
+irestart = 0          ! Read initial flow field ?
+icheckpoint = 100     ! Frequency for writing backup file
+ioutput = 100         ! Frequency for visualization
+ilist = 5             ! Frequency for the output
+nvisu = 1             ! Size for visualisation collection
+
+/End
+
+!=================
+&Statistics
+!=================
+
+nstat = 1             ! Size arrays for statistic collection
+
+/End
+
+!====================
+&PartiParam
```

Comment:
The documentation should be updated to describe the new input

Reply:
Yes we need to check this

Reply:
Updated